packages/flutter_test/lib/src/binding.dart

```diff
@@ -1226,24 +1226,52 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
 /// frames.
 ///
 /// These values are set on the binding's
-/// [LiveTestWidgetsFlutterBinding.framePolicy] property. The default is
-/// [fadePointers].
+/// [LiveTestWidgetsFlutterBinding.framePolicy] property.
+///
+/// {@template flutter.flutter_test.frame_policy}
+/// The default is [fadePointers]. Setting this to anything other than
+/// [onlyPumps] means pumping extra frames, which might involve calling builders
+/// more, or calling paint callbacks more, etc, which might interfere with the
+/// test. If you know your test file wouldn't be affected by this, you can set
+/// it to [fullyLive] or [benchmarkLive] persistently in that particular test
+/// file.
+///
+/// To set a value while still allowing the test
+/// file to work as a normal test, add the following code to your test file at
+/// the top of your `void main() { }` function, before calls to
+/// [testWidgets]:
+///
+/// ```dart
+/// TestWidgetsFlutterBinding binding = TestWidgetsFlutterBinding.ensureInitialized();
+/// if (binding is LiveTestWidgetsFlutterBinding) {
+///   binding.framePolicy = LiveTestWidgetsFlutterBindingFramePolicy.[thePolicy];
+/// }
+/// ```
+/// {@endtemplate}
 enum LiveTestWidgetsFlutterBindingFramePolicy {
-  /// Strictly show only frames that are explicitly pumped. This most closely
-  /// matches the behavior of tests when run under `flutter test`.
+  /// Strictly show only frames that are explicitly pumped.
+  ///
+  /// This most closely matches the [AutomatedTestWidgetsFlutterBinding]
+  /// (the defualt binding for `flutter test`) behavior.
   onlyPumps,
 
   /// Show pumped frames, and additionally schedule and run frames to fade
   /// out the pointer crosshairs and other debugging information shown by
   /// the binding.
   ///
+  /// This will schedule frames when pumped or when there has been some
+  /// activity with [TestPointer]s.
+  ///
   /// This can result in additional frames being pumped beyond those that
   /// the test itself requests, which can cause differences in behavior.
   fadePointers,
 
   /// Show every frame that the framework requests, even if the frames are not
   /// explicitly pumped.
   ///
+  /// The major difference with [benchmarkLive] is the latter ignores frame
```

Comment:
The major difference with what? fadePointers?
```
The major difference between [fadePointers] and [benchmarkLive] is that the latter ignores...
```
OR
```
The major difference between this policy and [benchmarkLive] is that the latter ignores...
```

Reply:
Done with the second.